LOCAL_DEV_SETUP.md

```diff
@@ -0,0 +1,263 @@
+# Local Development Setup for Simpler Grants.gov
+
+## Overview
+
+This document provides step-by-step instructions for setting up the Simpler Grants.gov repository locally using Docker Compose and a devcontainer.
+
+## Prerequisites
+
+- **Docker & Docker Compose** (version 20.10+)
+- **Git** for version control
+- **VS Code** (optional, but recommended for devcontainer support)
+- **VS Code Remote Development Extension** (if using devcontainer)
+
+## Quick Start
+
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/HHS-OCIO/simpler-grants-gov.git
+cd simpler-grants-gov
+```
+
+### 2. Start the Full Docker Compose Stack
+
+```bash
+docker compose up --build -d
+```
+
+This brings up all services:
+- **PostgreSQL** (port 5432) - Main database
+- **OpenSearch** (port 9200) - Search engine
+- **LocalStack** (port 4566) - Local S3 storage
+- **Mock OAuth2 Server** (port 5001)
+- **Mock SOAP Service** (port 8082)
+- **API** (port 8080) - Flask API service
+- **Frontend** (port 3000) - Next.js frontend
+- **Storybook** (port 6006) - Component library
+
+### 3. Initialize the Database
+
+```bash
+cd api
+make db-migrate              # Run migrations
+make setup-postgres-db       # Create schemas
+```
+
+### 4. Set Up OpenSearch (Search Index)
+
+Before loading data, ensure the OpenSearch index alias is created:
+
+```bash
+# The index and alias will be created when you populate search data
+# But for local dev testing, you can quickly verify the alias exists:
+curl http://localhost:9200/_cat/aliases?v
+```
+
+### 5. Load Data
+
+```bash
+# Populate database with test seed data
+make db-seed-local
+
+# Populate OpenSearch with opportunities and agencies
+make populate-search-opportunities
+make populate-search-agencies
+```
+
+## Configuration
+
+### Environment Variables
+
+Key environment files:
+- `api/local.env` - API configuration (copied from template)
+- `frontend/.env.local` - Frontend configuration (if needed)
+
+To customize for your local environment, create an `override.env` file in the `api/` directory:
+
+```bash
+cd api
+cp local.env override.env
+# Edit override.env to customize settings
+```
+
+### Using VS Code Devcontainer
+
+A `.devcontainer/devcontainer.json` is included (kept local via `.git/info/exclude`). To use it:
+
+1. Open the workspace in VS Code
+2. When prompted, select "Reopen in Container"
+3. VS Code will build and connect to the devcontainer with all dependencies installed
+
+The devcontainer automatically forwards ports and mounts the repository for live development.
+
+## Running Tests
```

Comment:
Tests are documented in https://github.com/HHS/simpler-grants-gov/blob/e2780af15c52d6a2c61fad6f3c15fc84b4f17fa0/api/README.md#running-tests-locally already